openapi/openapi.yaml

```diff
@@ -697,6 +697,66 @@ components:
       pattern: ^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$
       example: '1.0.1'
 
+    Tag:
+      description: 'Tag'
+      title: 'Tag'
+      type: object
+      properties:
+        name:
+          maxLength: 512
+          type: string
+        id:
+          description: 'Database ID for this object.'
+          readOnly: true
+          type: integer
+        url:
+          description: 'URL for this resource.'
+          readOnly: true
+          type: string
+          format: uri
+        related:
+          description: 'Data structure with URLs of related resources.'
+          readOnly: true
+          type: object
+          additionalProperties:
+            type: string
+        summary_fields:
+          description: >-
+            Data structure with name/description for related
+            resources.
+          readOnly: true
+          type: object
+          additionalProperties: {}
+        active:
```

Comment:
We don't have active and not active tags.

Reply:
Agree.

Reply:
updated